test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMParameterSpecTest.java

```diff
@@ -36,7 +42,12 @@
  */
 public class GCMParameterSpecTest {
 
-    private static final int[] IV_LENGTHS = { 96, 8, 1024 };
+    /*
+     * OpenSSL3 only supports IV lengths up to 16Bytes.
+     * When the IV length is set to be larger than 16Bytes, an error is thrown.
+     * According to the OpenSSL docs(https://www.openssl.org/docs/man1.1.1/man3/EVP_CIPHER_CTX_block_size.html), in OpenSSL1.1.1 and older, there is no error thrown but unpredictable behavior will happen for large IV sizes.
```

Comment:
Please split `16Bytes` as "16 bytes".
Line 48 is much too long: please wrap.

Reply:
Edited.